cmd/kubeadm/app/cmd/join.go

```diff
@@ -331,7 +331,7 @@ func (j *Join) Run(out io.Writer) error {
 		}
 
 		// run kubeadm init preflight checks for checking all the prequisites
-		glog.Infoln("[join] running pre-flight checks before initializing the new control plane instance")
+		fmt.Printf("[join] running pre-flight checks before initializing the new control plane instance\n")
```

Comment:
?? why?

Reply:
@Klaven 
it should be a `fmt.Println` instead of `fmt.Printf(.....\n)`. please fix this.

@timothysc 
in the case control plane join this `glog.Infoln()` would introduce the glog formatting with time / __FILE__, __LINE__, etc. i think Lucas' original idea was to have the default kubeadm output glog-free.
@chuckha can confirm.

Reply:
IIRC @chuckha had set a option to disable the glog premable.  
Right now I'd like to switch it all to glog and provide uniformity there.  

It's too late in the cycle to hit 1.12 w/cleanup work so we'll have to punt until 1.13 and try to provide better clarity on goals.  Ideally, we move it all to glog, strip out unnecessary details by default and provide more verbose logs with verbosity levels.

Reply:
^ +100